manifests/2.0.1/opensearch-dashboards-2.0.1.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Looks like you can remove this change from here and let bot take care of it: https://github.com/opensearch-project/opensearch-build/pull/2170
:)